pathfinding_service.rst

```diff
@@ -95,7 +95,10 @@ The arguments are POSTed as a JSON object.
 
 Returns
 """""""
-A list of path objects. A path object consists of the following information:
+
+A list of path objects and a feedback token when successful.
+
```

Comment:
just to be nitpicky but who knows what a path object is? Maybe we can find a more common term or describe that one?

Reply:
It's described ion the table below. I'll make that clearer.